packages/client/src/components/drive.folder.vue

```diff
@@ -118,7 +118,7 @@ function onDrop(ev: DragEvent) {
 
 	//#region ドライブのファイル
 	const driveFile = ev.dataTransfer.getData(_DATA_TRANSFER_DRIVE_FILE_);
-	if (driveFile != null && driveFile != '') {
+	if (driveFile !== null && driveFile !== '') {
```

Comment:
As for comparisons with null, `! =` and `! ==` treat `undefined` differently, so Misskey uses `! =` and `==` for null comparisons.
https://github.com/misskey-dev/misskey/blob/9230334a319c93d1604576778ed39c9de9d510ce/packages/client/.eslintrc.js#L21

Reply:
👍 I'll readjust it for the null checks then, I probably accidentally did this in multiple places. (because of the other reported issues in the same lines usually)

Reply:
This should be fixed now! Readjusted all null checks I was able to find and lint error count didn't increase back up again 😃